docs/source/platform-features/message-bus/rabbitmq/rabbitmq-ssl-auth.rst

```diff
@@ -14,7 +14,7 @@ configurations can be seen by running the following command:
 
 .. code-block:: bash
```

Comment:
Do we ever mention rabbitmq without auth?

Reply:
We used to do test rmq without ssl. But not since we got all the ssl certs in place

Reply:
I mean can volttron run rabbitmq without ssl?  If so then we should have an example?

Reply:
Can this be a separate issue/PR ?

Reply:
Certainly!  Make an issue reference it here and we can deal with that at that time.